bot/buttons.py

```diff
@@ -1,10 +1,14 @@
+from typing import List
+
 from telegram import InlineKeyboardButton
 from telegram import InlineKeyboardMarkup
 
+from database.types import User
+
 
-def get_user_buttons(users: list):
-    keyboard = list()
-    for user in users:
-        line = [InlineKeyboardButton(user['name'], callback_data=user['id'])]
-        keyboard.append(line)
+def get_user_buttons(users: List[User]) -> InlineKeyboardMarkup:
+    keyboard = [
+        [InlineKeyboardButton(user.name, callback_data=user.id)]
```

Comment:
I think we should use telegram username(update.effective_user.username) instead of just name